agent/src/beerocks/slave/son_slave_thread.cpp

```diff
@@ -1461,13 +1463,16 @@ bool slave_thread::handle_cmdu_platform_manager_message(
             platform_settings = response->platform_settings();
             wlan_settings     = response->wlan_settings();
 
+            auto db = AgentDB::get();
+
+            // Local copy on cuurent process database instance, to be removed on PPM-83 phase 5
```

Comment:
```suggestion
            // Local copy on the current process database instance, to be removed on PPM-83 phase 5
```

Reply:
This is an only a temporary comment. I'll remove this comment and its friends in the very near future. So don't be bothered by it.